src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -314,6 +314,8 @@ internal string DefaultNamespace
                        w => w.OnDefaultNamespaceChanged(Id, value));
         }
 
+        internal void SetMaxLangVersion(string version) => _workspace.SetMaxLanguageVersion(Id, version);
```

Comment:
Wondering if this should be a property (even set only!) so if anybody ever wants to make it public off of VisualStudioProject it's a property to stay consistent with everything else.